bundle/edu.gemini.phase2.skeleton.servlet/src/main/scala/edu/gemini/phase2/template/factory/impl/gnirs/GnirsSpectroscopy.scala

```diff
@@ -259,4 +259,12 @@ case class GnirsSpectroscopy(blueprint:SpGnirsBlueprintSpectroscopy, exampleTarg
     addNote("NOTE2: Bad pixels in XD mode") in TargetGroup
     addNote("NOTE3: Differential refraction and XD mode") in TargetGroup
   }
+
+  // requests 2020.04.08 Siyi
+  // change mean parallactic angle as the default set up for XD mode
+  if (xd == SXD || xd == LXD) {
+    val included = curIncludes(TargetGroup).toSet
+    val all      = Set(5,6,7,8,9,10,11,12,13,14,22,23)
+    forObs((all & included).toSeq: _*)(setPositionAngleConstraint(PosAngleConstraint.PARALLACTIC_ANGLE))
```

Comment:
This seems reasonable assuming the `all` set is correct.

Reply:
It was confirmed with Andy in the JIRA task.